test/modules/packages/Toml/Toml.chpl

```diff
@@ -46,7 +46,7 @@ Parser module with the Toml class for the Chapel TOML library.
 
 
    // Prints a line by line output of parsing process
-   config const DEBUG: bool = false;
+   config const debugToml: bool = false;
```

Comment:
I'd suggest either making this `debugTomlParser` (since it's specific to the parser), or moving it up to top-level module, and calling it `debugTOML`